docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+
+services:
+  app:
+    container_name: app
```

Comment:
The whole thing works perfectly 👍
The only thing I would change are the container names.

What do you think about `vimcolorschemes_app` and `vimcolorschemes_db`?

Reply:
Thank you for check. I agree with your change.
It also works well after the changes.
```
$ docker-compose ps
       Name                     Command             State            Ports          
------------------------------------------------------------------------------------
vimcolorschemes_app   npm run start::local          Up      0.0.0.0:8000->8000/tcp  
vimcolorschemes_db    docker-entrypoint.sh mongod   Up      0.0.0.0:27017->27017/tcp
```